esx_service/utils/auth_data.py

```diff
@@ -229,7 +235,7 @@ def replace_vms(self, conn, vms):
             )
 
             conn.executemany(
-                "INSERT INTO vms(vm_id, tenant_id) VALUES (?, ?)",
+                "INSERT INTO vms(vm_id, vm_name, tenant_id) VALUES (?, ?, ?)",
```

Comment:
nit : if it's really "update", then instead of DELETE then INSERT it could be a snigle INSERT OR REPLACE.
Does not really matter in our case , but generally a good practice to avoid unneeded rescans